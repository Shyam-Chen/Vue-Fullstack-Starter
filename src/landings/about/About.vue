<template>
  <div>About Page</div>
</template>

<script>
export default {
  metaInfo: {
    title: 'About | Oh My Vue',
  },
};
</script>
