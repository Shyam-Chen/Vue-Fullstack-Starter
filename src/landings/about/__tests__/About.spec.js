import { createLocalVue, shallowMount } from '@vue/test-utils';
// import I18n from 'vue-i18n';

import About from '../About';
// import en from '../_languages/en';

const localVue = createLocalVue();

// localVue.use(I18n);

describe('About', () => {
  let [wrapper/* , i18n */] = [];

  beforeEach(() => {
    // i18n = new I18n({
    //   locale: 'en',
    //   messages: { en: { home: en } },
    // });

    wrapper = shallowMount(About, {
      localVue,
      // i18n,
      mocks: {},
    });
  });

  it('should render an initial component', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
