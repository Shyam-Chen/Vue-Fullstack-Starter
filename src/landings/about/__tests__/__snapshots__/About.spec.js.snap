// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About should render an initial component 1`] = `<div>About Page</div>`;
